db/lock.go

```diff
@@ -113,17 +114,17 @@ func AcquireGlobalLock(id string) (bool, error) {
 	return acquired, err
 }
 
-// ReleaseGlobalLock relinquishes the global lock for the given id.
-func ReleaseGlobalLock(id string) error {
+// ReleaseLock relinquishes the lock for the given id.
+func ReleaseLock(id string) error {
 	session, db, err := GetGlobalSessionFactory().GetSession()
 	if err != nil {
 		return err
 	}
 	defer session.Close()
 
 	// will return mgo.ErrNotFound if the lock expired
-	return db.C(LockCollection).Update(
-		bson.M{"_id": GlobalLockId, "locked_by": id},
+	return db.C(lockCollection).Update(
+		bson.M{"_id": id},
 		bson.M{"$set": bson.M{"locked": false}},
```

Comment:
Why not add variables for each key in the lock model with bsonutil.MustHaveTag?

Reply:
I think the answer to your question is "legacy code".

Given that this is code with a short lease on life, I'm not inclined to change it, but we we could.